packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -137,6 +143,23 @@ export function ArticleMetadataPanel({
     }
   }, [data?.authors, authors, socialMediaAuthors])
 
+  const [createAuthor] = useCreateAuthorMutation({
+    refetchQueries: [getOperationNameFromDocument(AuthorListDocument)]
```

Comment:
Why did you add this? What is the purpose of that?

Reply:
I use it to make the new author I just created appear automatically in the list after clicking the button so the user can then select it without needing to refresh. Is there a better solution ?

Reply:
ahh. This is awesome. Good job